cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -1105,13 +1105,11 @@ func isErrorFeatureFlagNotEnabled(err error) bool {
 	// Tailscale control plane when a Tailscale Service API call is made for a
 	// tailnet that does not have the Tailscale Services feature flag enabled.
 	const messageFFNotEnabled = "feature unavailable for tailnet"
-	var errResp *tailscale.ErrResponse
-	ok := errors.As(err, &errResp)
-	return ok && strings.Contains(errResp.Message, messageFFNotEnabled)
+	return err != nil && strings.Contains(err.Error(), messageFFNotEnabled)
```

Comment:
Here it's a pointer if the error is non-nil- so perhaps just do the simplest thing and check the error message